### PR TITLE
update unit test related to intent check

### PR DESCRIPTION
**Pull request by plateaukao**
update unit tests related to intent handling

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -20,6 +20,13 @@ allprojects {
         google()
         jcenter()
     }
+
+    tasks.withType(Javadoc) {
```
Comment (plateaukao) [inline build.gradle:24]: fix build jcenter binary javaDoc task issue.

File: line-sdk/src/test/java/com/linecorp/linesdk/api/internal/LineApiClientImplTest.java [hunk 2]
```diff
@@ -106,7 +105,7 @@ public void testLogoutWithApiCallError() {
 
         assertSame(expectedResponse, actualResponse);
         verify(internalOauthApiClient, times(1)).revokeRefreshToken(CHANNEL_ID, ACCESS_TOKEN);
-        assertEquals(ACCESS_TOKEN, accessTokenCache.getAccessToken());
+        assertEquals(null, accessTokenCache.getAccessToken());
```
Comment (mutant0113) [inline line-sdk/src/test/java/com/linecorp/linesdk/api/internal/LineApiClientImplTest.java:108]: Suggest replacing this with
`assertNull(accessTokenCache.getAccessToken());`

File: line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java [hunk 3]
```diff
@@ -35,6 +34,9 @@
 import java.util.Collection;
 import java.util.List;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
```
Comment (mutant0113) [inline line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java:39]: I found these lines moved after rearranging imports. Suggest rearranging imports.

File: line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java [hunk 4]
```diff
@@ -200,8 +203,8 @@ public void testConvertToCompatibleIntent() throws Exception {
         assertNull(intentHolder.getStartActivityOptions());
 
         // Test for non new LINE and single target
-        setLineAppVersion("6.8.0");
```
Comment (mutant0113) [inline line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java:203]: Why do we need to remove this line? The comment above says we need to test `new LINE` and `non new LINE` test cases?

File: line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java [hunk 5]
```diff
@@ -211,7 +214,6 @@ public void testConvertToCompatibleIntent() throws Exception {
         assertFalse(intentHolder.isLineAppAuthentication());
 
         // Test for non new LINE and multiple target
-        setLineAppVersion("6.8.0");
```
Comment (mutant0113) [inline line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java:214]: ditto.

File: line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java (hunk 4 above)
Comment (plateaukao) [reply line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java:203]: Now, we do not exclude LINE version below 6.9.0 any more. I will check how to set it to null.

File: line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java (hunk 3 above)
Comment (plateaukao) [reply line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java:39]: it's the result after re-arrangement. :)

Comment (mutant0113) [reply line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java:39]: okay..it seems like my format is different from SDK's.

**Review by mutant0113 — APPROVED** [review]
LGTM after rephrasing the comments.

**Pull request merged**